tests/consensus_tests/test_named_vector_crud.py

```
@@ -0,0 +1,299 @@
+
+from .fixtures import create_collection, upsert_random_points
+from .test_shard_transfer_deferred import VECTOR_DIM
+from .utils import *
+
+N_PEERS = 3
+COLLECTION_NAME = "test_vector_crud"
+
+
+def create_vector_name(peer_url, collection_name, vector_name, config, timeout=10, wait=True):
+    """Create a named vector via PUT /collections/{name}/vectors/{vector_name}"""
+    r = requests.put(
```

Ruff (0.15.9)

[error] Probable use of `requests` call without timeout

(S113)